common/source/docs/common-holybro-kakutef4-mini.rst

```diff
@@ -0,0 +1,145 @@
+.. _common-holybro-kakutef4-mini:
+
```

Comment:
You should probably add that this board support bdshot by default

Reply:
done